src/namespaces/PlaybackNamespace.js

```diff
@@ -189,9 +195,8 @@ export default class PlaybackNamespace extends GMusicNamespace {
             // Make sure that this is the first of the notifications for the
             // insertion of the song information elements.
             if (!currentTrack.equals(lastTrack)) {
-              this.emit('change:track', currentTrack);
```

Comment:
Swapped it back, except I left a blank line deletion in by accident. I can add another commit to fix that if you want, but that might be a bit overkill...?